src/app/qgisapp.cpp

```diff
@@ -14721,7 +14765,7 @@ void QgisApp::addPluginToWebMenu( const QString &name, QAction *action )
   QMenu *menu = getWebMenu( name );
   menu->addAction( action );
 
-  // add the Vector menu to the menuBar if not added yet
+  // add the Web menu to the menuBar if not added yet
```

Comment:
was this change intended? Seems out of place but I may be wrong.

Reply:
Not related to the main issue, but yes, intended change.